consumer-goods/3.0/modules/ROOT/pages/cg-assets.adoc

```diff
@@ -70,7 +70,7 @@ Accelerator assets include API specifications, implementation templates, custom
 
 == See Also
 
-* xref:accelerators-home::getting-started.adoc[Getting Started Guide]
+* xref:accelerators-home::getting-started.adoc[Getting Started]
 * xref:accelerators-home::common-services.adoc[Accelerator Common Services]
 * xref:accelerators-home::shared-assets.adoc[Accelerator Shared Assets]
 * xref:prerequisites.adoc[Prerequisites]
```

Comment:
I know this isn't part of the review, but you could take this opportunity to change this to match our guidance (Before You Begin).

Reply:
I'll add this to the next work item, too.